portfolio/src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,30 @@
+ package com.google.sps.data;
+
+/** Class to hold the constants used in this project. */
+public final class Constants {
```

Comment:
FYI for capstone you will want to split your constants (and associated datastore and such) in their own packages.